mobile/src/Student/screens/Game/PhaseOneBasicGamePlay/index.js

```diff
@@ -319,17 +319,17 @@ const styles = StyleSheet.create({
         textAlign: 'center',
     },
     hintsViewCard: {
-        marginTop: -40,
-        paddingBottom: scale(20),
-        maxHeight: verticalScale(400),
+        marginTop: -scale(40),
```

Comment:
Same for this and the above change : D

Reply:
Realized there were some existing ones that were scale which should have been vertical it seems so made those corrections as well. Thank you!

Reply:
Aww perfect, thanks a lot! ❤️

Reply:
Thanks for your help!